library/log.inc

```diff
@@ -349,28 +349,28 @@ function sql_checksum_of_modified_row($statement)
 
    /* Framing sql statements for calculating checksum */
     if ($table == "form_physical_exam") {
```

Comment:
btw, ignore my advice above and do not escape table names below.
Except for maybe the last one. Let me look into this a bit since would like to avoid doing this because the logging engine is already a performance drag for OpenEMR and do not want to make it worse.
TODO

Reply:
To clarify, keep binding $rid like you already did, though.

Reply:
For the above TODO, best to not escape table here (note you haven't, so nothing for you to do here) or else the logging engine will get even slower.